operations/research-and-development/training.md

```diff
@@ -0,0 +1,35 @@
+# Recommended reading and resources
+
+## Developers
+
+- [Code Complete, Steve McConnell](https://www.amazon.com/Code-Complete-Practical-Handbook-Construction/dp/0735619670) - Best practices and guidelines for writing high quality code.
+- [Design Patterns, Erich Gamma, Richard Helm, Ralph Johnson and John Vlissides (aka "Group of Four")](https://www.amazon.com/Design-Patterns-Elements-Reusable-Object-Oriented-ebook/dp/B000SEIBB8) - Fundamental reading on design patterns. Other design pattern books work too, this is one of the most popular.
+
+## Product management
+
+Courses
+
+- [Harvard Business School PM 101](https://sites.google.com/site/hbspm101/home/2015-16-sessions/the-mrd-customer-discovery)
+
+## Software strategy
+
+- [Monetizing Open Source (Or, All Enterprise Software)](https://a16z.com/2017/04/10/monetizing-open-source-enterprise-software) - Required reading for business roles.
+
+## System security
+
+Papers and Course Materials
+
+- [Framework for Improving Critical Infrastructure Cybersecurity. National Institute of Standards and Technology](https://templatelab.com/cybersecurity-framework) - Standards for internal Mattermost security processes and safeguards.
+- [Computer Security in the Real World. Butler Lampson](https://research.microsoft.com/en-us/um/people/blampson/69-SecurityRealIEEE/69-SecurityRealIEEE.pdf) - Fundamental challenges with system security.
+- [Course notes from CS513: System Security (Cornell University). Fred B. Schneider](https://www.cs.cornell.edu/courses/cs513/2007fa/02.outline.html) - Well written introduction to system security from one of the leaders in the field.
+
+## Additional training resources
+
+Recommended training materials are recommended by role at three different levels of priority:
+
+- P1 - Required - Complete within 30 days of starting in role.
```

Comment:
I don't think we've ever enforced that new engineers read certain books within a timeframe of starting. Maybe we should update the language to make it clear that it's a recommendation and not required.